frontends/p4/toP4/toP4.cpp

```diff
@@ -1051,25 +1051,35 @@ bool ToP4::preorder(const IR::IfStatement* s) {
     visit(s->condition);
     builder.append(") ");
     if (!s->ifTrue->is<IR::BlockStatement>()) {
+        builder.append("{");
```

Comment:
Only actually necessary if s->ifTrue is an IR::IfStatement with ifFalse == nullptr?  I guess the redundant braces are not too ugly though (and are probably clearer even when ifFalse != nullptr)